src/Controls/tests/Core.UnitTests/Layouts/FlexLayoutTests.cs

```diff
@@ -133,6 +141,34 @@ public void UnconstrainedWidthChildrenHaveWidth()
 			Assert.Equal(100, flexFrame.Width);
 		}
 
+		[Fact]
+		public void FlexLayoutPaddingShouldBeAppliedCorrectly()
+		{
+			var padding = 16;
+			var root = new VerticalStackLayout();
+			
+			var boxView = new TestBoxView
+			{
+				Color = Colors.Red
+			};
+
+			var layout = new FlexLayout
+			{
+				Direction = FlexDirection.Column, 
+				BackgroundColor = Colors.LightGreen,
+				Padding = new Thickness(padding),
+				HeightRequest = 500,
+				WidthRequest = 500,
+				Children = { boxView }
+			};
+			root.Add(layout);
+			var measure = layout.CrossPlatformMeasure(500, 500);
+			layout.CrossPlatformArrange(new Rect(Point.Zero, measure));
+
+			var frame = layout.Children[0].Frame;
+			Assert.Equal(padding, frame.X);
```

Comment:
This is looking good!

Can we check the padding all around the child? Make sure that it's inset correctly from all sides and has the expected width/height?

Reply:
@PureWeen, I have updated the Unit cases based on your concerns, could you please validate once and let me know if there are any concerns.